main_test.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainHandlerWhenNoCity(t *testing.T) {
```

Comment:
В тесте пресутсвуют город, но не тот
```suggestion
func TestMainHandlerWhenWrongCity(t *testing.T) {
```